vyper/compile_lll.py

```diff
@@ -226,6 +226,20 @@ def compile_to_assembly(code, withargs=None, break_dest=None, height=0):
         o = compile_to_assembly(code.args[0], withargs, break_dest, height)
         o.extend(['PUSH1', MemoryPositions.FREE_VAR_SPACE, 'MSTORE', 'PUSH1', 32, 'PUSH1', MemoryPositions.FREE_VAR_SPACE, 'SHA3'])
         return o
+    # SHA3 a 64 byte value
+    elif code.value == 'sha3_64':
+        o = compile_to_assembly(code.args[0], withargs, break_dest, height)
+        o.extend([
+            'PUSH1', MemoryPositions.FREE_VAR_SPACE,
+            'MSTORE'
+        ])
+        o.extend(compile_to_assembly(code.args[1], withargs, break_dest, height))
+        o.extend([
+            'PUSH1', MemoryPositions.FREE_VAR_SPACE2,
+            'MSTORE',
+            'PUSH1', 64, 'PUSH1', MemoryPositions.FREE_VAR_SPACE, 'SHA3'
+        ])
+        return o
```

Comment:
@daejunpark would be great if you could double check this for me :smile:

Reply:
`sha3_64` looks good to me.

Reply:
Great thanks ;)